src/pages/login/GitHub.tsx

```diff
@@ -0,0 +1,53 @@
+import * as pages from "codeforlife/components/page"
+import { type FC, useEffect } from "react"
+import CflLogoImage from "../../images/cfl_logo.png"
+import GitHubLoginButton from "../../components/GitHubLoginButton"
+import { Image } from "codeforlife/components"
+import { useLoginWithGitHubMutation } from "../../api/session"
+
+export interface GitHubProps {}
+
+const GitHub: FC<GitHubProps> = () => {
+  const [loginWithGitHub] = useLoginWithGitHubMutation()
+
+  useEffect(() => {
+    const urlParams = new URLSearchParams(window.location.search)
+    const code = urlParams.get("code")
+
+    if (code) {
+      urlParams.delete("code")
+      window.history.replaceState(null, "", window.location.pathname)
+
+      loginWithGitHub({ code })
+        .unwrap()
+        .then(() => {
+          window.location.href = "/agreement-signatures"
+        })
+        .catch(err => {
+          window.location.href = "/login"
+          alert(`Login failed: ${err}`)
+        })
+    }
+  }, [loginWithGitHub])
+
+  return (
+    <pages.Page>
+      <pages.Section>
+        <div
```

Comment:
Delete this div. Use MUI's [Stack](https://mui.com/material-ui/react-stack/) instead. You can add spacing between element's using the Stack's spacing property.